src/examples/navbars/NavbarDefault.vue

```diff
@@ -184,8 +184,8 @@ watch(
               />
             </a>
             <div
-              class="dropdown-menu dropdown-menu-animation ms-n3 dropdown-md p-3 border-radius-xl mt-0 mt-lg-3"
-              aria-labelledby="dropdownMenuPages"
+              class="dropdown-menu ms-n3 dropdown-md p-3 border-radius-xl mt-0 mt-lg-3"
```

Comment:
`dropdown-menu-animation` caused weird spacing issues on the mobile menu. The spacing was more off-putting than the lack of animation, so I just removed it.

example without it removed:
![image](https://github.com/UniverseTBD/universeTBD.github.io/assets/22155312/d4a2605b-d665-400d-942a-65813fa6283a)


example with it removed:
![image](https://github.com/UniverseTBD/universeTBD.github.io/assets/22155312/f0a4c58e-59ce-4781-98d1-49a6842bb1a8)